src/math_eigen.h

```diff
@@ -240,7 +240,7 @@ namespace MathEigen {
               Jacobi<double,double*,double**,double const*const*>::
               SortCriteria sort_criteria =
               Jacobi<double,double*,double**,double const*const*>::
-              SORT_DECREASING_EVALS
+              DO_NOT_SORT
```

Comment:
Why this change?
Having consistently sorted eigenvalues would be a **welcome** property of the new implementation. If that means to reset a few of the regression tests, because they have slowly diverging results due to floating-point math is acceptable. The current implementation has a problem that it can result into differently ordered eigenvalues and vectors on different platforms, if some eigenvalues have a somewhat similar value.

Reply:
The original "jacobi()" function does not sort it's eigenvalues or eigenvectors.  I was just trying to more closely reproduce the previous code's behavior.  I just wanted to be as consistent as possible.  In the simulations I just ran, I am not sorting the eigenvalues.

Either way, users can override this default behavior by passing an additional argument to the "jacobi3()" function.  (The "sort_criteria" argument, 4th argument.)  If you like I can repeat those tests with the eigenvalues sorted and (if everything is okay) I can revert back to making this the default behavior.  Sound good?